solvebio/query.py

```diff
@@ -941,3 +949,300 @@ def execute(self, **params):
         _params.update(**params)
         response = self._client.post('/v2/batch_query', _params)
         return response
+
+
+class QueryFile(object):
```

Comment:
In hindsight, it looks like it would be simple to create a base class (QueryBase) as a mixin class with everything in QueryFile (except __init__, _clone, and _build_query). Everything looks exactly the same.

How about creating a `class QueryBase(object)` and inheriting that from `Query` and `QueryFile`?

Reply:
If you can recall, I suggested something similar to that, with a difference that QueryBase should have been an abstract class. However, when I think again it is not necessary to be the abstract class.

Reply:
Yea I remember. I just looked through and it seems that most of the methods are exactly the same, so you can just use a mixin class.